src/python/pants/option/global_options.py

```diff
@@ -65,6 +65,13 @@ def register_bootstrap_options(cls, register):
                       'pants.backend.python',
                       'pants.backend.jvm',
                       'pants.backend.codegen',
```

Comment:
No, it won't make it an error, it'll become an error on that version.

Pants itself is already not using those stubs, those deprecation warnings are for any custom pants tasks that might be using them.